### DISCUSSION TITLE
Formatting of text e.g. in 'Kopfbereich'

### DISCUSSION THREAD

**Discussion by rzlbrmft** [dragotin/kraft#201] [Q&A] [answered]
Is it possible to format the text in the 'Dokumentkopf' with tabs or at least blanks, e.g. in 'Einleitender Text auf der ersten Seite'?
When I format it like this in the field 'Einleitender Text auf der ersten Seite' 
Bearbeiter:                 abc
SAP Bestell-Nr.:         123
Firma:                         abc
it is shown in the 'Kopfbereich' and printed like this
Bearbeiter: abc
SAP Bestell-Nr.: 123
Firma: abc

![Screenshot_20230405_102450](https://user-images.githubusercontent.com/68049401/230024977-f0f8adde-a750-4475-930a-4f447e113ee0.png)

**Comment by noseshimself** [comment]
You *should* be able to enter HTML; if your printing template is getting that on the paper (see my examples in the distribution) this should work. It's not beautiful but it will do.

**Comment by dragotin** [comment, marked as answer]
Handled here: https://github.com/dragotin/kraft/issues/152
